.circleci/config.yml

```diff
@@ -1,22 +1,42 @@
 version: 2
 jobs:
-  one:
+  lint:
     docker:
       - image: circleci/node:8-browsers
```

Comment:
circleci runs all tests within docker containers which means you can actually run test tests in the exact same environment locally using their `circleci` CLI too.  You can also use the tool's `config validate` command to check that changes made to config.yml are valid.